apps/src/templates/instructions/ThreeColumns.jsx

```diff
@@ -22,7 +22,7 @@ const ThreeColumns = (props) => {
       width: '100%',
       position: 'relative',
       float: 'left',
-      overflowY: 'scroll',
+      overflowY: middleScrollable ? 'scroll' : undefined,
```

Comment:
Why aren't we just using [`overflow-y: auto`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y), which is supposed to do exactly what you want here?